src/Lib/ManticoreBackup.php

```diff
@@ -301,7 +301,7 @@ protected static function restoreState(FileStorage $storage, string $path): bool
 			$from = $file->getRealPath();
 			$to = dirname($storage->getOriginRealPath($from));
 			if (!is_dir($to)) {
-				FileStorage::createDir($to, $from);
```

Comment:
Static calls should use ClassName instead of dynamic variable value.
But probably, maybe `createDir` should not be static for better flexibility in supporting different storage engines in the future.